src/config.cpp

```diff
@@ -367,11 +367,16 @@ static int parse_channels(libconfig::Setting &chans, device_t *dev, int i) {
 					<<channel->freq_count<<" elements\n";
 				error();
 			}
-			if(chans[j].exists("squelch") && libconfig::Setting::TypeList == chans[j]["squelch"].getType() && chans[j]["squelch"].getLength() < channel->freq_count) {
```

Comment:
line 425 gives a warning that `squelch` is no longer supported so length check can be removed